packages/boot/src/boot.component.ts

```diff
@@ -0,0 +1,96 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/boot
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {
+  Context,
+  inject,
+  Constructor,
+  Binding,
+  BindingScope,
+} from '@loopback/context';
+import {
+  Component,
+  Booter,
+  BootOptions,
+  BOOTER_PHASES,
+  CoreBindings,
+  Application,
+} from '@loopback/core';
+import {resolve} from 'path';
+import {BootBindings} from './keys';
+
+import * as debugModule from 'debug';
+const debug = debugModule('loopback:boot:bootstrapper');
+
+export class BootComponent extends Context implements Component {
```

Comment:
I would rather leave `BootComponent` to be binder for boot related services. We can create `ApplicationBooter` as the coordinator.